etc/kayobe/inventory/group_vars/powerconnect-switches

```diff
@@ -0,0 +1,6 @@
+# Dict containing additional configuration for switches managed by the
+# genericswitch ML2 mechanism driver. For per-switch configuration of switches
+# in kolla_neutron_ml2_generic_switch_hosts, this may be set as a group or
+# host variable for the switch host.
+kolla_neutron_ml2_generic_switch_extra:
+  ngs_max_connections: 1
```

Comment:
This will conflict with the existing variable in etc/kayobe/inventory/group_vars/mgmt-switches, and we will lose the physnet setting (I think later in the alphabet wins).